pkg/operation/seed/seed.go

```diff
@@ -686,22 +686,16 @@ func BootstrapCluster(k8sGardenClient kubernetes.Interface, seed *Seed, config *
 
 // DesiredExcessCapacity computes the required resources (CPU and memory) required to deploy new shoot control planes
 // (on the seed) in terms of reserve-excess-capacity deployment replicas. Each deployment replica currently
-// corresponds to resources of (request/limits) 500m of CPU and 1200Mi of Memory.
-// ReplicasRequiredToSupportSingleShoot is 4 which is 2000m of CPU and 4800Mi of RAM.
-// The logic for computation of desired excess capacity corresponds to either deploying 2 new shoot control planes
-// or 3% of existing shoot control planes of current number of shoots deployed in seed (3 if current shoots are 100),
-// whichever of the two is larger
+// corresponds to resources of (request/limits) 2 cores of CPU and 6Gi of RAM.
+// This roughly corresponds to a single, moderately large control-plane.
+// The logic for computation of desired excess capacity corresponds to deploying 2 such shoot control planes.
+// This excess capacity can be used for hosting new control planes or newly vertically scaled old control-planes.
 func DesiredExcessCapacity(numberOfAssociatedShoots int) int {
```

Comment:
```suggestion
 func DesiredExcessCapacity() int {
```

This is an unused parameter.

Reply:
I kept it just in case we need it later. Will remove it :-)